packages/react-native/React/Base/RCTConvert.m

```diff
@@ -1487,6 +1487,105 @@ + (NSPropertyList)NSPropertyList:(id)json
     integerValue)
 
 #if TARGET_OS_OSX // [macOS
+// Used to convert the JSON from `role` to the corresponding
+// NSAccessibilityRole (NSString*), has slightly different mappings than the
```

Comment:
I think we don't need to mention `Used to convert the JSON from `role` to the corresponding
// NSAccessibilityRole (NSString*)` since that's the purpose of this whole file. The rest about "slightly different mappings" seems more important to have first